go/vt/mysqlctl/mysqld.go

```diff
@@ -1063,7 +1064,7 @@ socket=%v
 `, connParams.Uname, connParams.Pass, connParams.UnixSocket)
 	}
 
-	tmpfile, err := os.CreateTemp("", "example")
+	tmpfile, err := os.CreateTemp("", "defaults-extra-file-")
```

Comment:
Is the trailing dash here intentional?

Reply:
yes - `os.CreateTemp()` then appends a random number to that.